bootstrap/kubeadm/api/v1alpha3/condition_consts.go

```diff
@@ -1,68 +0,0 @@
-/*
```

Comment:
Can you check in a follow-up PR if we can drop upstreamv1beta1?

I think we only needed these types for Kubernetes < v1.15 and just kept them because these types were embedded in v1alpha3

Reply:
Opened  a WIP PR - I think we should be able to do it.